orchestrator/clickhouse/root.go

```diff
@@ -74,13 +88,21 @@ func New(r *reporter.Reporter, configuration Configuration, dependencies Depende
 		return nil, fmt.Errorf("resolutions need to be configured, including interval: 0")
 	}
 
+	c.d.Daemon.Track(&c.t, "orchestrator/clickhouse")
+
 	return &c, nil
 }
 
 // Start the ClickHouse component.
 func (c *Component) Start() error {
 	c.r.Info().Msg("starting ClickHouse component")
 
+	// stub to prevent tomb dying immediately after migrations are done
+	c.t.Go(func() error {
+		<-c.t.Dying()
+		return nil
+	})
```

Comment:
Yes, that's expected. Nothing much to do, this is part of the design of the tombs.

Reply:
I think they have an explanation here: https://github.com/go-tomb/tomb/issues/16 (but right now, I don't really grasp the why).